proto/protovalidate/buf/validate/validate.proto

```diff
@@ -156,6 +156,8 @@ message MessageRules {
   //      silently ignored when unmarshalling, with only the last field being set when
   //      unmarshalling completes.
   //
+  // Note that adding a field to a `oneof` will also set the IGNORE_IF_UNPOPULATED on the fields. This means
+  // only the field that os set will be validated and the unset fields are not validated according to the field rules.
```

Comment:
// only the field that os set

should be "is set"

Reply:
Has since been fixed here: https://github.com/bufbuild/protovalidate/pull/383